tests/bluetooth/bsim_bt/bsim_test_l2cap_stress/src/main.c

```diff
@@ -372,6 +386,11 @@ static void test_central_main(void)
 	LOG_DBG("*L2CAP STRESS Central started*");
 	int err;
 
+	/* Prepare tx_data */
+	for (size_t i = 0; i < sizeof(tx_data); i++) {
+		tx_data[i] = (uint8_t)i;
```

Comment:
The type of the variable being assigned to (`tx_data[i]`) is already uint8_t, so truncation happens either way, ie the explicit typecast is redundant.

Reply:
at least with the catch it's visible that it's on purpose and not a mistake